packages/experimental/Menu/src/Menu/useMenu.ts

```diff
@@ -9,8 +9,10 @@ export const useMenu = (props: MenuProps): MenuState => {
   const isSubmenu = context.triggerRef !== null;
   const isControlled = typeof props.open !== 'undefined';
   const [open, setOpen] = useMenuOpenState(isControlled, props);
+  const openOnHover = isSubmenu;
 
   return {
+    openOnHover,
     ...props,
     open,
```

Comment:
I think if we do it this order and `props` contains a key named `openOnHover`, it will overwrite the value you provide on like 15. Is that intentional?

Reply:
See answer to comment above